### PR TITLE
api-docs: add all "legacy" wallet RPC commands

### PR DESCRIPTION
**Pull request by pinheadmz**
split off from https://github.com/handshake-org/handshake-org.github.io/pull/41

### CODE REVIEW

File: src/api-docs/source/includes/_wallet_rpc.md [hunk 1]
```diff
@@ -0,0 +1,2858 @@
+# RPC Calls - Wallet
+
+```shell--curl
+# examples in these docs will use a DIFFERENT environment variable
```
Comment (boymanjor) [inline src/api-docs/source/includes/_wallet_rpc.md]: nit: capitalization.

File: src/api-docs/source/includes/_wallet_rpc.md [hunk 2]
```diff
@@ -0,0 +1,2858 @@
...
+```json
+null
+```
+
+Back up wallet database and files to directory created at specified path.
+
+### Params
+N. | Name | Default |  Description
+--------- | --------- | --------- | -----------
+1 | path | Required | Path and name of destination directory for backup
```
Comment (boymanjor) [inline src/api-docs/source/includes/_wallet_rpc.md]: suggestion: "Path and name" here may be confusing. Perhaps "Path to backup's destination directory."

File: src/api-docs/source/includes/_wallet_rpc.md [hunk 3]
```diff
@@ -0,0 +1,2858 @@
...
+# End of dump
+```
+
+Creates a new human-readable file at specified path with all wallet private keys in 
+Wallet Import Format (base58).
+
+### Params
+N. | Name | Default |  Description
+--------- | --------- | --------- | -----------
+1 | path | Required | Path and name of destination file for dump
```
Comment (boymanjor) [inline src/api-docs/source/includes/_wallet_rpc.md]: ditto

**Comment by pinheadmz** [comment]
@boymanjor thanks for reviewing all these! I know its a lot.
Fixed the nits and rebased on master to fix conflicts in the index.

**Pull request merged**